content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,213 @@
+
+```shell
+kubectl create -f https://k8s.io/examples/application/job/cronjob.yaml
+```
+Keluaran akan mirip dengan ini:
+
+```
+cronjob.batch/hello created
+```
+
+Atau kamu dapat menggunakan `kubectl run` untuk membuat sebuah CronJob tanpa menulis sebuah konfigurasi yang lengkap:
+
+```shell
+kubectl run hello --schedule="*/1 * * * *" --restart=OnFailure --image=busybox -- /bin/sh -c "date; echo Hello from the Kubernetes cluster"
+```
+
+Setelah membuat sebuah CronJob, untuk mengecek statusnya kalian dapat menggunakan perintah berikut:
+
+```shell
+kubectl get cronjob hello
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        <none>          10s
+```
+
+Seperti yang kamu lihat dari hasil perintah di atas, CronJob belum menjadwalkan atau menjalankan pekerjaan apa pun.
+Mengamati Job yang akan dibuat sekitar satu menit:
+
+```shell
+kubectl get jobs --watch
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME               COMPLETIONS   DURATION   AGE
+hello-4111706356   0/1                      0s
+hello-4111706356   0/1           0s         0s
+hello-4111706356   1/1           5s         5s
+```
+
+Sekarang kamu telah melihat satu Job berjalan yang dijadwalkan oleh "hello" CronJob.
+Kamu dapat berhenti mengamati Job dan melihat CronJob lagi untuk melihat CronJob menjadwalkan sebuah Job:
+
+```shell
+kubectl get cronjob hello
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        50s             75s
+```
+
+Kamu dapat melihat bahwa CronJob `hello` berhasil menjadwalkan sebuah Job pada waktu yang ditentukan dalam `LAST SCHEDULE`. Saat ini ada 0 Job yang aktif,  berarti sebuah Job telah selesai atau gagal.
+
+Sekarang, temukan Pod yang dibuat oleh jadwal Job terakhir dan melihat keluaran bawaan dari salah satu Pod
+
+{{< note >}}
+Nama Job dan nama Pod itu berbeda.
+{{< /note >}}
+
+```shell
+# Ganti "hello-4111706356" dengan nama Job di sistem kamu
+pods=$(kubectl get pods --selector=job-name=hello-4111706356 --output=jsonpath={.items[*].metadata.name})
+```
+Menampilkan log sebuah Pod:
+
+```shell
+kubectl logs $pods
+```
+Keluaran akan mirip dengan ini:
+
+```
+Fri Feb 22 11:02:09 UTC 2019
+Hello from the Kubernetes cluster
+```
+
+## Menghapus sebuah CronJob
+
+Ketika kamu tidak membutuhkan sebuah CronJob lagi, kamu dapat megnhapusnya dengan perintah `kubectl delete cronjob <cronjob name>`:
+
+```shell
+kubectl delete cronjob hello
+```
+
+Menghapus CronJob akan menghapus semua Job dan Pod yang telah terbuat dan menghentikanya dari pembuatan Job tambahan.
+Kamu dapat membaca lebih lanjut tentang menghapus Job di [_garbage collection_](/docs/concepts/workloads/controllers/garbage-collection/).
+
+## Menulis Speifikasi Sebuah Cron
+
+Seperti semua konfigurasi Kubernetes, sebuah CronJob membutuhkan bidang `apiVersion`, `kind`, dan `metadata`. Untuk informasi
```

Comment:
I used to keep _field_ word, but this is a nice translation "bidang", what all other think?
cc @kubernetes/sig-docs-id-reviews

Reply:
any update regarding this mas?

Reply:
I second the idea of keping _binding_ for now.

Reply:
So I'll change to use field word